packages/gafl-webapp-service/src/pages/renewals/renewal-inactive/renewal-inactive.njk

```diff
@@ -2,11 +2,11 @@
 {% from "fieldset/macro.njk" import govukFieldset %}
 {% from "back-link/macro.njk" import govukBackLink %}
 
-{% if data.reason === 'not-due' %}
+{% if data.reason === data.reasonCodes.NOT_DUE %}
     {% set title = mssgs.renewal_inactive_title_1 %}
-{% elseif data.reason === 'expired' %}
+{% elseif data.reason === data.reasonCodes.EXPIRED %}
     {% set title =  mssgs.renewal_inactive_title_2 %}
-{% elseif data.reason === 'not-annual' %}
+{% elseif data.reason === data.reasonCodes.NOT_ANNUAL %}
     {% set title = mssgs.renewal_inactive_title_3 %}
 {% endif %}
```

Comment:
I actually meant for this to be put into the getData as well - so `title` would be part of the data object, eliminating this structure from the template